src/test/ui/rfc-0107-bind-by-move-pattern-guards/feature-gate.no_gate.stderr

```diff
@@ -0,0 +1,9 @@
+error[E0008]: cannot bind by-move into a pattern guard
```

Comment:
Oh, I guess it would be nice (when on nightly and with MIR-borrowck enabled) if this issued a diagnostic suggesting the feature gate.

Reply:
(done, see commit c329897)